src/main/resources/jupyter/google_sign_in.js

```diff
@@ -0,0 +1,38 @@
+require.config({
+    "shim": {
+        "gapi": {
+            "exports": "gapi"
+        }
+    },
+    "paths": {
+        "gapi": "https://apis.google.com/js/platform"
+    }
+})
+
+/// TEMPLATED CODE
```

Comment:
Please add a short description of Leo templating here, to give someone reading this file on its own an idea where to look.

Reply:
👍